README.md

```diff
@@ -1,2 +1,170 @@
-# Math-Magicians-V2
-This is a new version of my "Math-Magicians" project where I create an SPA with a calculator and a quote generator.
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Math magicians V2</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] Remove this link as it points to a section which has been removed from the readme file making it a broken link